src/main.rs

```diff
@@ -42,6 +46,11 @@ struct BlockState {
 
 type ServerRef = Arc<Server>;
 
+lazy_static! {
```

Comment:
There's no reason why this needs to be lazy_static. Just load it in main and feed the data forward to the Server struct there.